### DISCUSSION TITLE
high cpu mysql

### DISCUSSION THREAD

**Discussion by ajeje989** [AzuraCast/AzuraCast#5244] [Q&A]
Hi
today i update from 0.14.1 to 0.15.2 ( stable version with docker )

but with top i have high mysql ?

check the screen

https://prnt.sc/XiJaSZVZf6N8

this is a screen

if i check mariadb logs

docker-compose logs -f mariadb
https://prnt.sc/l5aQfIlmNeJ7

this is a screen redis logs

https://prnt.sc/d0968OF9lAAj

is normal ?

i have dedicated server with 8 core with 32 gb ram .

i think i need to fix mysql.

**Comment by BusterNeece** [comment]
@ajeje989 We already have a GitHub issue that you created a few days ago for this specific problem, no need to make a discussion as well.
